tools/ios/MASTG-TOOL-0137.md

```diff
@@ -0,0 +1,35 @@
+---
+title: GlobalWebInspect
+platform: ios
+source: https://github.com/ChiChou/GlobalWebInspect
+host:
+- ios
+---
+
+!!! warning
+
+    This tool may or may not work depending on your macOS / iOS combination.
+
+GlobalWebInspect can be installed on a jailbroken iOS device to enable web-inspection on any WebView in any application. The tweak can be installed by copying it over to your device and using `sudo dpkg -i <file>`.
+
+To activate the web inspection you have to follow these steps:
+
+1. On the iOS device open the Settings app: Go to **Safari -> Advanced** and toggle on _Web Inspector_.
```

Comment:
isn't this a new TECH that could be used in TESTs?

Reply:
It can probably be used for some dynamic analysis on the webview tests.